pkg/cmd/server/start/start_allinone.go

```diff
@@ -105,16 +113,16 @@ func NewCommandStartAllInOne(basename string, out io.Writer) (*cobra.Command, *A
 	BindListenArg(listenArg, flags, "")
 	BindImageFormatArgs(imageFormatArgs, flags, "")
 
-	startMaster, _ := NewCommandStartMaster(basename, out)
-	startNode, _ := NewCommandStartNode(basename, out)
-	startNodeNetwork, _ := NewCommandStartNetwork(basename, out)
-	startEtcdServer, _ := NewCommandStartEtcdServer(RecommendedStartEtcdServerName, basename, out)
+	startMaster, _ := NewCommandStartMaster(basename, delegatedOutput)
```

Comment:
These ought to remain as `out`, since those commands are essentially this command.